float32[] LastWPpassed
float32[] MyLoc4LastWP

